modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/raft/RocksDbClusterStateStorage.java

```diff
@@ -86,6 +87,13 @@ public RocksDbClusterStateStorage(Path dbPath, String nodeName) {
 
     @Override
     public CompletableFuture<Void> start() {
+        // delete existing data, relying on log playback
+        try {
+            RocksDB.destroyDB(dbPath.toString(), options);
+        } catch (RocksDBException e) {
+            return failedFuture(new CmgStorageException("Unable to drop RocksDb storage", e));
+        }
+
         init();
```

Comment:
This is where it doesn’t work out well, because init() throws an `CmgStorageException`, but the `return failedFuture` should be, please fix it.